src/components/tabs/index.tsx

```diff
@@ -2,7 +2,7 @@ import {
   KeyboardEvent,
   KeyboardEventHandler,
   ReactElement,
-  useLayoutEffect,
+  useEffect,
```

Comment:
Tabs were being rendered on the server (due to ISR I think?) and so a bunch of warnings were printing in the console. Lmk the best course here, I'm happy to leave it, but the layout effect doesn't work on server rendering.

Reply:
(oops, crossed wires there - didn't see your comment Ashlee! I trust ya'll so am good with either 👍)

Reply:
@zchsh I had made the comment earlier, but had to wait to submit until GitHub wasn't sad anymore. So no worries! You posted your comment first, I'm not sure why mine is injected into history like this. 😆

So one clarification because my comment was short: `useEffect` is okay to use here because the code in it that depends on the DOM being populated won't run unless `needToFocusNewElement.current` is `true`. It's only set to `true` when the keys that change focus are handled `onKeyUp`, and those events can only be triggered after the DOM has been populated.